scripts/manual_operations/manual_submission.py

```diff
@@ -57,29 +54,16 @@ def get_location_and_rb_from_database(database_client, run_number):
         print("Database not connected")
         return None
 
-    db_connection = database_client.connect()
-    location_query = f"""
-                    SELECT file_path
-                    FROM reduction_viewer_datalocation
-                    WHERE reduction_run_id = {run_number}
-                    """
-    location_result = db_connection.execute(location_query).fetchall()
-    if len(location_result) == 0:
-        return None
-
-    rb_query = f"""
-                SELECT reduction_viewer_experiment.reference_number
-                FROM reduction_viewer_reductionrun
-                    JOIN reduction_viewer_experiment
-                    ON reduction_viewer_reductionrun.experiment_id = reduction_viewer_experiment.id
-                WHERE reduction_viewer_reductionrun.id = {run_number}
-                """
+    reduction_run_records = database_client.get_reduction_run(instrument, run_number)
 
-    rb_result = db_connection.execute(rb_query).fetchall()
-    if len(rb_result) == 0:
+    if not reduction_run_records:
         return None
 
-    return location_result[0].file_path, rb_result[0].reference_number
+    reduction_run_record = reduction_run_records.order_by('run_version').first()
+    data_location = reduction_run_record.data_location.first().file_path
+    experiment_number = reduction_run_record.experiment.reference_number
```

Comment:
Your first usage of `reduction_run_record` (ln 63) calls `first()`
Your second usage (ln 64) doesn't. Is this correct?

Reply:
Also, it is a bit difficult at first glance to distinguish `reduction_run_records` and `reduction_run_record`.
Do you think it's worth naming one a bit differently? e.g. just `run_record` for the singular?